Mathlib/Algebra/Algebra/Hom.lean

```diff
@@ -236,7 +236,7 @@ end
 theorem id_apply (p : A) : AlgHom.id R A p = p :=
   rfl
 
-/-- Composition of algebra homeomorphisms. -/
+/-- Composition of algebra homomorphisms. -/
```

Comment:
While we're at it
```suggestion
/-- Composition of algebra homomorphisms -/

Reply:
Is that right? I'm looking at a bunch of docstrings right now and they all end in full stops.